£Lerror_reason¢DtypeEvalueYÅHTTPSConnectionPool(host='archive.ics.uci.edu', port=443): Max retries exceeded with url: /ml/datasets.html?format=&task=reg&att=mix&area=other&numAtt=10to100&numIns=100to1000&type=mvar&sort=attTypeUp&view=list (Caused by ConnectTimeoutError(<urllib3.connection.VerifiedHTTPSConnection object at 0x000000000F611630>, 'Connection to archive.ics.uci.edu timed out. (connect timeout=2)'))Ihttp_code¢DtypeEvalueÛQdownload_complete¢DtypeEvalueı